vkcs/provider.go

```diff
@@ -49,6 +52,45 @@ func (c *config) GetRegion() string {
 	return c.Region
 }
 
+// GetTenantID is implementation of getTenantID method
+func (c *config) GetTenantID() string {
+	return c.TenantID
+}
+
+var tempUrl = map[string]string{
+	"alerting":  "https://mcs.mail.ru/infra/alerting/v1/",
+	"templater": "https://mcs.mail.ru/infra/templater/v2/",
```

Comment:
any chance to put it in keystone catalog?

Reply:
done